build/simp-doc.spec

```diff
@@ -107,7 +107,7 @@ Buildarch: noarch
 Requires: links
 %if 0%{?el6}
 BuildRequires: centos-release-scl
-BuildRequires: python27
+BuildRequires: python36
```

Comment:
It looks like EL6 only has `python33` not `python36` so I don't think this will work properly.

Reply:
Reference: http://mirror.centos.org/centos/6/sclo/x86_64/rh/